index.html

```diff
@@ -46,6 +46,157 @@ <h2>I'm a software Developer</h2>
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+        </div>
+      </section>
+      <section class="aboutme">
+        <div class="text-container">
```

Comment:
Kindly align this container's elements with flex as this is part of the requirements and also a best practice :necktie: :heavy_check_mark:

Reply:
Done